README.md

```diff
@@ -133,6 +133,8 @@ describe('Testing MyComponent', () => {
 
 ## Example
 
+_The following example is a minimal setup of `react-a11y-dialog`. Additionally, you will need to add the required styles as per the recommendations in the `a11y-dialog` [styling docs](https://a11y-dialog.netlify.app/usage/styling)._
```

Comment:
After working on the AgnosticUI react dialog wrapper around `react-a11y-dialog` I'm realizing it might be useful to also point out that these styles need to not only be defined in a stylesheet, but also pushed in via the `classNames` object. I just can't think of how to say this in a brief and and eloquent way 🤔 

For example, if I have the CSS defined:

```css
.dialog,
.dialog-overlay {
  position: fixed;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
}

.dialog {
  z-index: 1001;
  display: flex;
}

.dialog[aria-hidden="true"] {
  display: none;
}

/* and more relevant CSS classes of course */
```

Those won't do anything until I also map the to `classNames` and pass:

```js
  const myClassNames = {
    container: styles.dialog,
    // etc.
  };
 
  return <A11yDialog classNames={myClassNames} ...otherstuff />
```

I realize this is on the one hand very basic and easy to figure out for an experienced developer. But again, just thinking of a lot of beginner/intermediates that could get tripped up and our goal is to get as many folks making these dialogs accessible as possible 😄 

Let me know if you think I'm overthinking this `classNames` thing, or, if you have any ideas and I'd be happy to do another iteration.

Reply:
I think maybe we could add a piece of information like:

> How you integrate these styles depends on the styling layer you chose for your project. However you apply these recommended styles is left at your discretion (classes, inline styles, CSS-in-JS…), for as long as they end up being applied to the elements being rendered by React.